Formula/heartbeat.rb

```diff
@@ -0,0 +1,86 @@
+class Heartbeat < Formula
+  desc "Lightweight Shipper for Uptime Monitoring"
+  homepage "https://www.elastic.co/products/beats/heartbeat"
+  url "https://github.com/elastic/beats/archive/v5.6.4.tar.gz"
+  sha256 "c06f913af79bb54825483ba0ed4b31752db5784daf3717f53d83b6b12890c0a4"
+
+  head "https://github.com/elastic/beats.git"
+
+  depends_on "go" => :build
+
+  def install
+    gopath = buildpath/"gopath"
+    (gopath/"src/github.com/elastic/beats").install Dir["{*,.git,.gitignore}"]
+
+    ENV["GOPATH"] = gopath
+
+    cd gopath/"src/github.com/elastic/beats/heartbeat" do
+      system "make"
+      libexec.install "heartbeat"
+
+      (etc/"heartbeat").install("heartbeat.yml")
+      (etc/"heartbeat").install("heartbeat.full.yml")
+      (etc/"heartbeat").install("heartbeat.template.json")
+      (etc/"heartbeat").install("heartbeat.template-es2x.json")
+      (etc/"heartbeat").install("heartbeat.template-es6x.json")
+    end
+
+    (bin/"heartbeat").write <<~EOS
+      #!/bin/sh
+        exec #{libexec}/heartbeat \
+        -path.config #{etc}/heartbeat \
+        -path.home #{prefix} \
+        -path.logs #{var}/log/heartbeat \
+        -path.data #{var}/lib/heartbeat \
+        $@
+    EOS
+  end
+
+  def post_install
+    # Make sure runtime directories exist
+
```

Comment:
You don't need the blank line, and the comment isn't really needed, since that's always the point of making directories in post install, but it doesn't hurt either, I suppose.